### PR TITLE
Add German translations (from Crowdin)

### PR DESCRIPTION
**Pull request by Blazzycrafter**
## Changes

Write here about the changes you've made

## Why 

Write here why you think this should be merged

## Checkmarks

- [ ] The modified functions have been tested.
- [ ] Used the same indentation as the rest of the project.
- [ ] Updated documentation (changelog).

## Additional information

If you have any additional information, write it here

### CODE REVIEW

**Comment by Blazzycrafter** [comment]
WTF

File: webroot/js/language.js
Comment (RainyXeon) [inline webroot/js/language.js:1]: Available language should in alphabetically

**Review by RainyXeon — CHANGES_REQUESTED** [review]
Do some change before merge

File: webroot/js/language.js
Comment (Blazzycrafter) [reply webroot/js/language.js:1]: not really my fault but sure let me fix that...
image

Comment (Blazzycrafter) [reply webroot/js/language.js:1]: or may its my f idk ... but i fix it

Comment (RainyXeon) [reply webroot/js/language.js:1]: Okay

Comment (Blazzycrafter) [reply webroot/js/language.js:1]: should be fixed now

Comment (RainyXeon) [reply webroot/js/language.js:1]: Lgtm

**Review by RainyXeon — APPROVED** [review]

**Pull request merged**